client/apps/Admin/src/components/SingleAssignmentForm.tsx

```
@@ -0,0 +1,237 @@
+        }
+
+        setSubmitError(null);
+        createMutation.mutate({
+            userId: selectedUserId,
+            taskId: selectedTaskId,
+        });
+    }
+
+    return (
+        <div className="modal-overlay" onClick={onClose}>
+            <div className="modal single-assignment-form" onClick={(e) => e.stopPropagation()}>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)